mlflow/diviner.py

```diff
@@ -318,6 +418,17 @@ def log_model(
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
     :param kwargs: Additional arguments for :py:class:`mlflow.models.model.Model`
+                   Additionally, for models that have been fit in Spark, the following supported
+                   configuration options are available to set.
+                   Current supported options:
+
+                   - `partition_by`  for setting a (or several) partition columns as a list of \
+                   column names. Must be a list of strings of grouping key column(s).
```

Comment:
```suggestion
                   - `partition_by`  for setting a (or several) partition columns as a list of
                      column names. Must be a list of strings of grouping key column(s).
```

I think we can remove `\`.

Reply:
we have to use the line continuation for rst lists or it will put the entries all on a single line.